collections/schema.go

```diff
@@ -161,6 +161,12 @@ func NewSchemaFromAccessor(accessor func(context.Context) store.KVStore) Schema
 	}
 }
 
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
```

Comment:
We could create a wrapper like in ORM and add those two methods there, then add a `GenesisHandler()`  method, but I do not think it has any benefits.

Reply:
I think it's fine this way 👍